pkg/raft/tracker/inflights_test.go

```diff
@@ -112,76 +94,61 @@ func TestInflightFreeTo(t *testing.T) {
 	in.FreeLE(0)
 
 	wantIn0 := &Inflights{
-		start: 1,
-		count: 9,
 		bytes: 945,
 		size:  10,
 		buffer: inflightsBuffer(
-			//          ↓------------------------
-			[]uint64{0, 1, 2, 3, 4, 5, 6, 7, 8, 9},
-			[]uint64{100, 101, 102, 103, 104, 105, 106, 107, 108, 109}),
+			//       ↓------------------------
```

Comment:
Can drop these pointer comments now since they all point at the first entry. Same below and above.

Reply:
will do in a followup. TFTR!